common/pom.xml

```diff
@@ -8,13 +8,26 @@
         <version>1.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
+         
+    <repositories>
+         <repository>
+            <id>opencollab-snapshot-repo</id>
+            <url>https://repo.opencollab.dev/maven-snapshots/</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
 
     <dependencies>
         <dependency>
             <groupId>org.geysermc</groupId>
             <artifactId>connector</artifactId>
             <version>1.2.0-SNAPSHOT</version>
-            <scope>provided</scope>
+            <scope>compile</scope>
```

Comment:
Why this change? We don't want to include the entire Geyser copy in GeyserSkinManager because it's always provided for us, if we use it.

Reply:
Oh I'll change it back